internal/clearingway/handlers.go

```diff
@@ -320,12 +328,11 @@ func (c *Clearingway) InteractionCreate(s *discordgo.Session, i *discordgo.Inter
 				fmt.Printf("Invalid custom ID received: \"%v\"\n", customID)
 				return
 			}
-
 			switch CommandType(command[1]) {
 			case CommandMenu:
-				// send_encounter_menu(command[2])
+				c.MenuEncounterSend(s, i, command[2])
```

Comment:
eh i shouldve addressed it in the first PR but we should probably be more specific on what `command[1]` and `command[2]` are. it might help initializing them earlier then passing those variables through here